app/code/Magento/Sales/Model/Service/CreditmemoService.php

```diff
@@ -3,11 +3,12 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
 
 namespace Magento\Sales\Model\Service;
 
 /**
- * Class CreditmemoService
```

Comment:
I think we should not remove the Class CreditmemoService in comment. You can add description below the class comment.